### PR TITLE
Epic: Token locking

### PR DESCRIPTION
**Pull request by schmanu**
## What this epic is about
This epic transforms the Safe DAO governance app into the Safe {Pass} | {DAO} app.

Overall the app is split in 2 parts:
- Activity & Rewards
- Governance & Claiming

While the second remains mostly the same the Activity & Rewards page is completely new.
It offers
- Locking Safe tokens to boost points
- A Leaderboard of top Safe token lock amounts
- A page to unlock & withdraw locked tokens
- Information about the Safe{Pass} Program

The Activity & Rewards part of the app is usable as Safe App and via WalletConnect as a standalone application.

### CODE REVIEW

File: pages/boost.tsx
Comment (usame-algan) [inline pages/boost.tsx:1]: Is this page still being used? The link "More about boost" seems to link to an external help article.

Comment (schmanu) [reply pages/boost.tsx:1]: No it can be deleted. Will delete it quickly. I forgot to push the deletion.

File: src/hooks/useSafeTokenBalance.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  const chainId = useChainId()
+  const address = useAddress()
+
+  return useSWR(web3 && address ? QUERY_KEY : null, (): Promise<[BigNumber, BigNumber, number, number] | undefined> => {
+    if (!address || !web3) {
+      return Promise.resolve([BigNumber.from(0), BigNumber.from(0), 0, 0])
+    }
+    return fetchLockedAmount(chainId, address, web3)
+  })
+}
```
Comment (usame-algan) [inline src/hooks/useSafeTokenBalance.ts:62]: I think this is unused

File: src/utils/lock.ts [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+  }
+}
+
+/**
+ * Receives the data for a specific Unlock.
+ *
+ * @param index index of the unlock
+ * @returns Unlock data for index: [amount, unlockedAt (in seconds)] or undefined
+ */
+export const fetchUnlockData = async (
```
Comment (usame-algan) [inline src/utils/lock.ts:117]: Also seems unused

**Review by usame-algan — APPROVED** [review]
Only minor nits. Looks good otherwise!

**Pull request merged**